contact_info_service.py

```
@@ -0,0 +1,141 @@
+                    "state",
+                    "website_or_email",
+                    "phone",
+                    "source_urls",
+                ],
+                "additionalProperties": False,
+            },
+            "strict": True,
+        }
+    }
+
+    def __init__(self, openai_client):
```

Ruff (0.14.1)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)